File: URI/Java/src/URI/_1020.java
package URI;

import java.util.Scanner;

public class _1020 {
    
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int dias = sc.nextInt();
        int anos = dias / 365;
        dias = dias % 365;
        int meses = dias / 30;
        dias = dias % 30;
        
        System.out.println(anos + " ano(s)");
        System.out.println(meses + " mes(es)");
        System.out.println(dias + " dia(s)");
    }
}

File: URI/1020.cpp
#include<iostream>
using namespace std;

int main(){
	int dias, anos, meses;

	cin >> dias;

	anos = dias / 365;
	dias -= anos * 365;

	meses = dias / 30;
	dias -= meses * 30;

	cout << anos << " ano(s)" << endl;
	cout << meses << " mes(es)" << endl;
	cout << dias << " dia(s)" << endl;

	return 0;
}